src/tax/display-settings.md

```diff
@@ -61,6 +61,18 @@ Showing prices that include and exclude tax can be confusing to the customer. To
    - `Including Tax`
    - `Including and Excluding Tax`
 
+1. For **Display Gift Wrapping Prices**, choose one of the following:
```

Comment:
Need to add icon for Commerce only option/step.

```suggestion
1. {:.ee-only}For **Display Gift Wrapping Prices**, choose one of the following:
```

Reply:
Done!